cyences_app_for_splunk/appserver/static/cs_device_inventory_table.js

```diff
@@ -209,7 +213,7 @@ require([
             | eval _time=strftime(latest_time, "%F %T") 
             | eval Select="CHECKBOX_THIS_".uuid 
             | rename ips as ip, hostnames as hostname, mac_addresses as mac_address
-            | table uuid, Select, _time, ip, hostname, mac_address, lansweeper_id, lansweeper_state, lansweeper_asset_type, lansweeper_os, lansweeper_user, lansweeper_description, qualys_id, QUALYS_OS, qualys_network_id, tenable_uuid, tenable_os, nessus_uuid, nessus_os, sophos_uuid, sophos_type, sophos_os, sophos_user, sophos_login_via, sophos_health, sophos_product_installed, crowdstrike_id,kaspersky_collected_by,kaspersky_version,kaspersky_host, kaspersky_status windows_defender_host 
+            | table uuid, Select, _time, ip, hostname, mac_address, lansweeper_id, lansweeper_state, lansweeper_asset_type, lansweeper_os, lansweeper_user, lansweeper_description, qualys_id, QUALYS_OS, qualys_network_id, tenable_uuid, tenable_os, nessus_uuid, nessus_os, sophos_uuid, sophos_type, sophos_os, sophos_user, sophos_login_via, sophos_health, sophos_product_installed, crowdstrike_id,kaspersky_collected_by,kaspersky_version,kaspersky_host, kaspersky_status, windows_defender_host, splunk_host, splunk_os, splunk_user, splunk_version, splunk_forwarder_type, splunk_server
```

Comment:
Do we need splunk_server?

What's the value for splunk_user? Is it multi-valued field?

Reply:
splunk_server is not much useful as we have hostname hence removing it.

I forgot to remove splunk_user from here as no such info available. I'll remove it.